u4-databases/features/uni-controller.js

```diff
@@ -0,0 +1,54 @@
+const { db } = require('../_services/firebase-initialized')
+
+const iterationLists = async (req, res) => {
+  const query = db.collection('iterations').get()
+  const iteration = (await query).docs.map(doc => doc.data())
```

Comment:
ตัวแปรนี้เป็นอาร์เรย์ที่มีการวนซ้ำหลายครั้ง ดังนั้นฉันขอแนะนำให้ใช้ชื่อพหูพจน์: `iterations`
This variable is an array containing multiple iterations, so I recommend using a plural name: `iterations`